src/common.c

```diff
@@ -32,3 +32,26 @@ void adc_clear_flag(uint32_t adc_peripheral, uint32_t flag)
 {
 	ADC_SR(adc_peripheral) = ~flag;
 }
+
+/**
+ * @brief Return a random integer in the [from, to] interval.
+ *
+ * The result includes both end points (`from` and `to`).
+ *
+ * Randomness is very poor and should only be used for low values. It is based
+ * on the cycle counter register, so in order to get really "random" values
+ * some user-dependent or environment-dependent delay is expected before it is
+ * used.
+ *
+ * @param[in] from The lowest possible value.
+ * @param[in] to The highest possible value.
+ *
+ * @return A random integer in the [from, to] interval.
+ */
+int32_t random_int(int32_t from, int32_t to)
```

Comment:
Why not use standard library? 
https://stackoverflow.com/questions/822323/how-to-generate-a-random-int-in-c

Reply:
You are right, fixed! :+1: